docs/src/main/asciidoc/build-analytics.adoc

```diff
@@ -0,0 +1,156 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Build Analytics
+:categories: analytics
+:summary: This guide presents what build analytics is and how to configure it.
+
+The Quarkus team has limited knowledge, from Maven download numbers, on the excellent growth of Quarkus and the amount of users reporting issues/concerns, but we are really flying blind on which platforms, OS, java combinations, build tools etc.
+The build analytics tool aims to provide us with this information.
+
+== How does it work?
+
+*Collection happens only at build-time* and not at runtime. Collection occurs during usage of Quarkus build tooling (maven, gradle). Your built application will not contain any data collection code.
```

Comment:
```suggestion
*Collection happens only at build-time* and not at runtime. Collection occurs during usage of Quarkus build tooling (Maven, Gradle). Your built application does not contain any data collection code.
```

Reply:
You can do it that way. It's not terribly important, but, in technical documentation, we [avoid the future tense for various reasons](https://www.google.com/search?q=avoid+future+tense+in+technical+writing&oq=avoid+future+tense+in+technical+writing&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQRRhAMgYIAhBFGEDSAQg4NzI4ajFqN6gCALACAA&sourceid=chrome&ie=UTF-8).

Reply:
Thanks @rolfedh. Will follow your guidance.